src/ftxui/screen/screen.cpp

```diff
@@ -191,7 +188,7 @@ void Screen::Print() {
 /// @param x The character position along the x-axis.
 /// @param y The character position along the y-axis.
 wchar_t& Screen::at(int x, int y) {
-  return PixelAt(x, y).character;
+  return PixelAt(x, y).character[0];
```

Comment:
Is it any useful to acess *the first character* in a screen cell? It would probably be better to have a *getTextAt*/*setTextAt* pair of functions.

Reply:
Before removing a function, I would have like to look into every projects depending on FTXUI and submit PR to avoid breaking them without notice.